src/core/p5.Graphics.js

```diff
@@ -60,6 +60,13 @@ p5.Graphics = class extends p5.Element {
 
     if (r === constants.WEBGL) {
       this._renderer = new p5.RendererGL(this.canvas, this, false);
+      const gl = this._renderer.GL;
+      const maxTextureSize = gl.getParameter(gl.MAX_TEXTURE_SIZE);
```

Comment:
This theoretically shouldn't change over time, so maybe we can add a method to RendererGL to be something like this:
```js
maxTextureSize() {
  if (!this._maxTextureSize) {
    this._maxTextureSize = this.GL.getParameter(this.GL.MAX_TEXTURE_SIZE);
  }
  return this._maxTextureSize;
}
```
`getParameter` calls are generally a bit slow, so we want to cache their results (or avoid calling them if there's a convenient way to do so.)